server/app/src/main/java/com/google/heartrate/wearos/app/sensors/HeartRateSensorListener.java

```diff
@@ -0,0 +1,98 @@
+package com.google.heartrate.wearos.app.sensors;
+
+import android.content.Context;
+import android.hardware.Sensor;
+import android.hardware.SensorEvent;
+import android.hardware.SensorEventListener;
+import android.hardware.SensorManager;
+import android.util.Log;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Listener for heart rate sensor.
+ */
+public class HeartRateSensorListener implements SensorEventListener {
+    private static final String TAG = HeartRateSensorListener.class.getSimpleName();
+
+    /** Sensor manager. */
+    private final SensorManager mSensorManager;
```

Comment:
Let's be consistent with the rest of your code: mSensorManager -> sensorManager?